### DISCUSSION TITLE
CityEnergyAnalyst v3.31.0

### DISCUSSION THREAD

**Discussion by shanshanhsieh** [architecture-building-systems/CityEnergyAnalyst#3166] [Announcements]
In this release, we fix the bug on GUI to import own geometries as input.
Additionally, building floor heights information is now handled in a consistent manner throughout the code.

From the change log:
- 2022-05-10 - 3.30.0 - #3151 how-to-review-a-pull-request
- 2022-05-10 - 3.30.0 - #3154 Remove hardcoded floor height from calculations
- 2022-04-27 - 3.30.0 - #3155 Fix import input files on dashboard
- 2022-04-27 - 3.30.0 - #3152 release-3.30.0

<hr /><em>This discussion was created from the release <a href='https://github.com/architecture-building-systems/CityEnergyAnalyst/releases/tag/v3.31.0'>CityEnergyAnalyst v3.31.0</a>.</em>

**Comment by shanshanhsieh** [comment]
Hi all, I will recommend switching to this version to have the full GUI functionalities!
Let us know how it works : )

**Reply by kholody** [reply]
error installing last version CEA 3.31.0 #3245

Hi. I tried to use this version as you recommend but I couldnot as follow<

Describe the bug
I have this issue more than once in the last step of installation although i have installed it before successfully so please help me as iam in the middle of a project
To Reproduce
Steps to reproduce the behavior:

Go to '..instalation page.'
Click on '.installing eith all the steps ...'
Scroll down to '.kindly find the screenshot and script of error down here ...'
See error
Expected behaviour
A clear and concise description of what you expected to happen.
Output folder: C:\Users\Kholod\Documents\CityEnergyAnalyst
Created wheel for cityenergyanalyst: filename=cityenergyanalyst-3.31.0-py3-none-any.whl size=32535379 sha256=f8a9924283eea64f96408ee45d4d89a282ff41359d3179fdc0976e3c1b85bf9f
Stored in directory: C:\Users\Kholod\AppData\Local\Temp\pip-ephem-wheel-cache-zz52qo23\wheels\1e\0c\25\f876e03c646d363a56092d5ce3c7396839e72a6f476723fae9
Successfully built cityenergyanalyst
Installing collected packages: cityenergyanalyst
Successfully installed cityenergyanalyst-3.31.0
pip install cityenergyanalyst==3.31.0 returned 0
"cea --help" returned 259
Installation failed - see Details

Screenshots
If applicable, add screenshots to help explain your problem.
Hardware (please complete the following information):

OS and version: [e.g. iOS 10.7 or Windows 10]
Version of CEA you are using [e.g. v. CEA 3.31.0]
![image](https://user-images.githubusercontent.com/98927433/192378392-f2ee40b6-ebae-4e28-be4c-565e9f2b5899.png)

**Reply by kholody** [reply]
hope to hear from you so soon 

with all the respect to your effort
